lib/signaling/v2/localparticipant.js

```diff
@@ -171,6 +176,16 @@ class LocalParticipantV2 extends LocalParticipantSignaling {
     }
     return publication;
   }
+
+  /**
+   * Updates the verbosity of network quality information.
+   * @param {NetworkQualityConfiguration} config
+   * @returns {void}
+   */
+  setNetworkQualityConfiguration(config) {
```

Comment:
@syerrapragada Can you make this more robust? You can use the pattern used in `lib/encodingparameters.js` and create a class called `NetworkQualityConfigurationImpl` which emits a "changed" event.